antiplag/models.py

```diff
@@ -51,3 +51,7 @@ def __str__(self):
     @property
     def matches(self):
         return len(self.matched_docs)
+
+    @property
+    def intervals(self):
+        return self.matched_docs[0]["intervals"]
```

Comment:
Property `intervals` of Result only returns intervals of first matched document? I don't get the logic behind it. What about other matched_docs in the list?

Reply:
fixed. returning intervals for each doc separately.